src/lib.rs

```diff
@@ -82,6 +82,11 @@ enum FeatureSupported<'a> {
 
 #[derive(Clone, Debug, PartialEq)]
 enum Query<'a> {
+    /// Return the attached state of the indicated process.
+    // FIXME the PID only needs to be optional in the
+    // non-multi-process case, which we aren't supporting; but we
+    // don't send multiprocess+ in the feature response yet.
+    Attached(Option<u64>),
```

Comment:
So once we fix the feature response to advertise multiprocess support this can lose the `Option` wrapper?

Reply:
I think so.  The only real difficulty is if we want to support clients that don't use this support.

I tend to think we shouldn't bother, but I guess it's worth checking the RSP-enabled strace (other than gdb and lldb, that's the only client I know of) before fully committing.